README.md

```diff
@@ -1,2 +1,17 @@
-# rhdh-installer
-Contains Helm Charts for installing GitOps/Pipeline Operators as well as installation scripts
+# AI RHDH Installer
+
+## Installation
+
+Run `helm upgrade --install <release-name> <path-to-chart>` to deploy default installations of the necessary operators.
+
+#### Example Install
+
+`helm upgrade --install setup-default-operators ./chart`
+
+## Troubleshooting
```

Comment:
```suggestion
### Uninstall Release
```

Reply:
I'm only thinking that this is something like a removal guide with clear instructions, so I'd prefer `Uninstall release` or something similar which sounds less negative. But is very minor.